WebApiBackend/WebApiBackend/Helpers/DevelopmentDatabaseSetup.cs

```diff
@@ -8,19 +8,161 @@ namespace WebApiBackend.Helpers
 {
     public class DevelopmentDatabaseSetup
     {
+        User user1, user2, user3;
+        Payment payment1, payment2;
+        // Payment[NAME]1 is for electricity Payment[NAME]2 is for Rent due to many to many relationship
+        UserPayment userPaymentYin1, userPaymentYin2, userPaymentBryan1,
+            userPaymentBryan2, userPaymentTeresa1, userPaymentTeresa2;
+        Schedule schedule1;
+        Flat flat1;
+
         private readonly FlatManagementContext _database;
 
+        public void InitialiseTestDataObjects()
+        {
+            user1 = new User
+            {
+                UserName = "YinWang",
+                FirstName = "Yin",
+                LastName = "Wang",
+                DateOfBirth = new DateTime(1994, 12, 23),
+                PhoneNumber = "0279284492",
+                Email = "YinWang@qq.com",
+                MedicalInformation = "N/A",
+                Password = "MustacheMan22"
+            };
+
+            user2 = new User
+            {
+                UserName = "TreesAreGreen",
+                FirstName = "Teresa",
+                LastName = "Green",
+                DateOfBirth = new DateTime(1996, 02, 12),
+                PhoneNumber = "0228937228",
+                Email = "GreenTrees@Yahoo.com",
+                MedicalInformation = "Vegan, Gluten-Free, Lactose Intolerant",
+                Password = "TreestheBest"
+            };
+
+            user3 = new User
+            {
+                UserName = "BeboBryan",
+                FirstName = "Bryan",
+                LastName = "Ang",
+                DateOfBirth = new DateTime(1984, 02, 09),
+                PhoneNumber = "02243926392",
+                Email = "ZianYangAng@Gmail.com",
+                MedicalInformation = "COVID-19, Extra Chromosome",
+                Password = "StolenGirlfriend123"
+            };
+
+            payment1 = new Payment
+            {
+                Id = 1,
+                PaymentType = PaymentType.Electricity,
+                Amount = 175,
+                Fixed = false,
+                Frequency = Frequency.Monthly,
+                StartDate = new DateTime(2020, 03, 07),
+                EndDate = new DateTime(2020, 06, 07),
+            };
+            payment2 = new Payment
+            {
+                Id = 2,
+                PaymentType = PaymentType.Rent,
+                Amount = 1000,
+                Fixed = true,
+                Frequency = Frequency.Monthly,
+                StartDate = new DateTime(2020, 03, 01),
+                EndDate = new DateTime(2020, 10, 01),
+            };
+            userPaymentBryan1 = new UserPayment
+            {
+                Payment = payment1,
+                User = user1,
+                UserName = user1.UserName,
```

Comment:
Should this be user3 instead of user1?

Reply:
😮 Yes it should, ill change that

Reply:
You changed the wrong one hahah, switch lines 82-83 with 97-98

Reply:
Oh i thought I changed Bryan, I’ll do that asap

Reply:
@mitchfaulcon I changed it, can you double check that I havent missed out on anything else